### PR TITLE
fix: api v3 login links

### PR DESCRIPTION
**Pull request by peterklingelhofer**
This should update the rest of the links for logging in with APIv3.

### CODE REVIEW

File: beta/src/stores/app.js [hunk 1]
```diff
@@ -252,7 +252,7 @@ function app () {
 
       if (redirect) {
         emitter.emit('redirect', {
-          dest: '/login',
+          dest: '/api/v3/user/connect/resonate',
```
Comment (auggod) [inline beta/src/stores/app.js]: You need the full url instead of the relative one so the redirect function uses window.location after defined time out.

**Comment by auggod** [comment]
I really missed that we had so many.

**Comment by peterklingelhofer** [comment]
> I really missed that we had so many.

No worries @auggod. Solved the merge conflicts, seems like you handled the part you commented on about the window.location redirect link. Good to go now?

**Comment by auggod** [comment]
No, I didn't, my last PR was only about the logout event. I fixed the PR title, it should have been logout not login event.

**Pull request merged**